src/api/java/com/minecolonies/api/research/IGlobalResearch.java

```diff
@@ -53,26 +48,43 @@
 
     /**
      * Start the research.
-     *
-     * @param player            the player starting it.
      * @param localResearchTree the local research tree to store in the colony.
      */
-    void startResearch(@NotNull final PlayerEntity player, @NotNull final ILocalResearchTree localResearchTree);
+    void startResearch(@NotNull final ILocalResearchTree localResearchTree);
 
     /**
      * Human readable description of research.
-     *
      * @return the description.
      */
     String getDesc();
 
+    /**
+     * Subtitle description of research.
+     * @return the optional subtitle name.
+     */
+    String getSubtitle();
+
     /**
      * Getter of the id of the research.
-     *
      * @return the String id.
      */
     String getId();
 
+    /**
+     * Getter of the research icon's resource location.
+     * @return the string of the ResourceLocation of the icon, in the pattern namespace:directory:(optional count).
+     */
+    String getIcon();
```

Comment:
That one can possibly include pseudo-ItemStack data, including item counts, or a texture location.  I'll change the jdoc to be clearer on that, but it can't be a ResourceLocation.

((Also currently under debate if it should even be used; the current commit has drawing of research icons disabled.))

Reply:
Then make it something like IIcon and add a parser which can parse the json spec into the IIcon object used for rendering.

But leaving this as a string is just asking for problems